---
layout: default
---

I lead research projects at the intersection of machine learning and experimental biology.

Currently, I'm a lead scientist at [Deep Genomics](https://deepgenomics.com).

## Links

* [CV](eli_bogart_cv.pdf)
* Email: eli at elibogart dot net
* [LinkedIn](www.linkedin.com/in/eli-bogart-7689b698)
* [Github](http://github.com/ebogart)
* [Twitter](http://twitter.com/eli_bogart) (never updated)

## Previous experience

Before joining Deep Genomics, I developed machine learning models of
transcriptional regulation at [CAMP4 Therapeutics](https://camp4tx.com).
Previously, at Kintai Therapeutics (now part of
[Senda Biosciences](https://sendabiosciences.com)), I
developed methods and pipelines for metagenomic data analysis, cloud
computing infrastructure, and visualization tools; grew the
computational biology team from 1 to 3 FTE; and led our efforts to
strategically apply machine learning techniques to accelerate the drug
discovery process.

In my academic work, I developed [computational tools to help
understand microbial communities and their impact on human
health](http://github.com/gerberlab/mitre) as a postdoc in the lab of
[Georg Gerber](http://gerber.bwh.harvard.edu) at Brigham and Women's
Hospital, and [nonlinear genome-scale optimization
models](http://github.com/ebogart/fluxtools) of [metabolism in C4
plants](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0151722)
during my PhD thesis research with [Chris
Myers](http://cbsu.tc.cornell.edu/staff/myers/) at Cornell University.

## Publications
* __Bogart, E.__ , R. Creswell, and G. K. Gerber (2019). [MITRE: inferring features from microbiota time-series data linked to host status.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1788-y) _Genome Biology_ 20:186.

* Bucci, V., B. Tzen, N. Li, M. Simmons, T. Tanoue, __E. Bogart__,
L. Deng, V. Yeliseyev, M. L. Delaney, Q. Liu, B. Olle, R. R. Stein,
K. Honda, L. Bry, and G. K. Gerber (2016). [MDSINE: Microbial
dynamical systems inference engine for microbiome time-series
analyses.](http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0980-6)
_Genome Biology_ 17(1):121.

* Allegretti, J. R., S. Kearny, N. Li, __E. Bogart__, K. Bullock,
G. K. Gerber, L. Bry, C. B. Clish, E. Alm, and J. R. Korzenik (2016).
Recurrent _Clostridium difficile_ infection associates with distinct
bile acid and microbiome profiles. _Alimentary Pharmacology and
Therapeutics_ 43(11):1142-53.

* __Bogart, E__. and C. R. Myers (2016). [Multiscale metabolic
modeling of C4 plants: connecting nonlinear genome-scale models to
leaf-scale metabolism in developing maize
leaves.](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0151722)
_PLoS ONE_ 11(3): e0151722.

* Franck, C., W. Ip, A. Bae, N. Franck, __E. Bogart__, and T. T. Le
(2008). Contact-mediated cell-assisted cell proliferation in a model
eukaryotic single-cell organism: an explanation for the lag phase in
shaken cell culture.  _Phys. Rev. E_ 77: 041905


