---
layout: default
---

I build tools to extract useful information from complex biological data.

Currently, I'm a senior scientist at [Kintai Therapeutics](http://kintaitx.com).

## Previous research experience

As a postdoc in the lab of [Georg
Gerber](http://gerber.bwh.harvard.edu) at Brigham and Women's
Hospital, I developed computational tools to help understand microbial
communities and their impact on human health.  My primary project was
[a method to infer interpretable predictive models of health outcomes
from human microbiome time series
data](http://github.com/gerberlab/mitre). I also worked on models
for the growth of engineered microbial consortia and connections
between the microbiome and bile acid levels in _C. difficile_
infection.

My PhD thesis research, with [Chris
Myers](http://cbsu.tc.cornell.edu/staff/myers/) at Cornell University,
focused on genome-scale models of metabolism in C4 plants. I created
[software to construct and solve flux balance analysis problems with
nonlinear constraints](http://github.com/ebogart/fluxtools), a new
reconstruction of the metabolic network of _Zea mays_, an algorithm
for inferring fluxes in metabolic models from transcriptomic data, and
a method to simulate evolution-like paths through high-dimensional
phenotypic fitness landscapes.

## Publications
* __Bogart, E.__  and G. K. Gerber (2018). MITRE: predicting host status from microbiota time-series data. Submitted; [available on bioRxiv.](https://www.biorxiv.org/content/early/2018/10/18/447250)

* Bucci, V., B. Tzen, N. Li, M. Simmons, T. Tanoue, __E. Bogart__,
L. Deng, V. Yeliseyev, M. L. Delaney, Q. Liu, B. Olle, R. R. Stein,
K. Honda, L. Bry, and G. K. Gerber (2016). [MDSINE: Microbial
dynamical systems inference engine for microbiome time-series
analyses.](http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0980-6)
_Genome Biology_ 17(1):121.

* Allegretti, J. R., S. Kearny, N. Li, __E. Bogart__, K. Bullock,
G. K. Gerber, L. Bry, C. B. Clish, E. Alm, and J. R. Korzenik (2016).
Recurrent _Clostridium difficile_ infection associates with distinct
bile acid and microbiome profiles. _Alimentary Pharmacology and
Therapeutics_ 43(11):1142-53.

* __Bogart, E__. and C. R. Myers (2016). [Multiscale metabolic
modeling of C4 plants: connecting nonlinear genome-scale models to
leaf-scale metabolism in developing maize
leaves.](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0151722)
_PLoS ONE_ 11(3): e0151722.

* Franck, C., W. Ip, A. Bae, N. Franck, __E. Bogart__, and T. T. Le
(2008). Contact-mediated cell-assisted cell proliferation in a model
eukaryotic single-cell organism: an explanation for the lag phase in
shaken cell culture.  _Phys. Rev. E_ 77: 041905

* __Bogart, E__. and C. R. Myers. Genome-scale modeling of the
  evolutionary path to C4 photosynthesis. _In preparation_.

## Links

* [CV](eli_bogart_cv.pdf)
* [Github](http://github.com/ebogart)
* [Twitter](http://twitter.com/eli_bogart)
* Email: eli at elibogart dot net

